README.md

```diff
@@ -2511,19 +2511,95 @@ ari.sounds.list()
 
 ### Events
 
-Event listeners can be registered on the client as well as on resource instances. Note that resource instance listeners are tied to the instance while client listeners receive all events of a given type regardless of which resource the event is for.
+Event listeners can be registered on the client as well as on resource instances.
+
+#### Client Events
+
+Client events are received for all events of a given type regardless of which resource the event is for.
+
+```javascript
+ari.on('StasisStart', function (event, channelInstance) {
+    // will be called for all channels that enter the Stasis application
+});
+```
+
+#### Resource Instance Events
+
+Resource instance events are only received for the given type and resource.
+
+Callbacks:
+
+```javascript
+var channel = ari.Channel();
+channel.on('StasisStart', function (event, channelInstance) {
+    // will only be called when the channel above enters the Stasis application
+});
+
+channel.originate(
+  {endpoint: 'PJSIP/endpoint', app: 'applicationName'},
+  function (err, channelInstance) {}
+);
```

Comment:
channel.originate() is what creates the channel. StasisStart will then fire and the event listener will be called since the StasisStart is for the channel that was just created.